modules/nw-metallb-configure-l2-advertisement-interface.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-with-L2-advertisement-interface_{context}"]
+= Configuring MetalLB with a L2 advertisement by using interface
+
+By default, the IP address from `IPAddressPool` that has been assigned to the service, is advertised from all the network interfaces. The `interfaces` field in the `L2Advertisement` custom resource definitions is used to restrict those network interfaces which advertise the `IPAddressPool`.
+
+This example shows how to configure MetalLB so that the `IPAddressPool` is advertised only from the network interfaces listed in the `interfaces` field of all nodes.
```

Comment:
Could the instances of `IPAddressPool` in the module intro be changed to "IP address pool", like is shown in step 1?

Reply:
that too works.